doc/usecases/record_elektra/UC_ansible-libelektra_start_recording.md

```diff
@@ -0,0 +1,18 @@
+# Use Case: Start session recording after Ansible run
+
+## Summary
+
+- **Title:** Start session recording after Ansible run
+- **Scope:** Recording
+- **Level:** User Goal
+- **Actors:** User (usually sysadmin)
+- **Brief:** After a new elektra configuration has been rolled out via Ansible, automatically enable session recording
```

Comment:
```suggestion
- **Brief:** After a new elektra configuration has been rolled out via Ansible, automatically enable or reset session recording for the appropriate namespaces
```

Reply:
What would 'the appropriate namespaces' be?

Reply:
Probably all of them which had changes (any recording happened)?